CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 > Any trouble, please visit: https://github.com/diogocavilha/fancy-git#troubleshooting-pick
 
+
+### v6.2.9
+
+- Add styling to git stash listing  for clarity
+
+### v6.2.8
```

Comment:
I added the change from the previous update on centralizing config files to the `changelog`. Followed the consistency from the logs in `v5....`.
So this new update would come in as a `v.6.2.9` and so on.

Reply:
That's right.